mlir/utils/jenkins/Jenkinsfile

```diff
@@ -190,15 +192,20 @@ def rebootNode() {
 void build_fixedE2ETests(String codepath) {
     // Limit the number of lit workers to 8 for navi21 codepath on nightly CI as a workaround for issue#702
     limit_lit_workers = false
+    // Filter out dense_output_bf16.mlir for navi32 codepath as a workaround for issue#1023
+    filter_out_tests = false
     if ( (codepath == 'navi21') && (params.nightly == true) ) {
```

Comment:
Do you need `navi31` here as well?

Reply:
(We might also want to drop this just being on nightly?)

Reply:
I'm dropping this test for both navi31 and navi32, although I could have done this only for navi32. The comment is indeed confusing. We will drop this for nightly as well.